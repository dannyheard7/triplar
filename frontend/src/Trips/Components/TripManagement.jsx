import React from "react";
import {Link} from "react-router-dom";

export default function TripManagement(props) {
    let tripId = props.trip.id;

    return (
        <div className="trip-management">
            <Link to={`/trips/${tripId}/edit`}><button className="btn btn-light">Edit</button></Link>
            <Link to={`/trips/${tripId}/delete`}><button className="btn btn-danger">Delete Trip</button></Link>
        </div>
    )
}
