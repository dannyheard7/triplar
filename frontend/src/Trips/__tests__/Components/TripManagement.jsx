import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import TripManagement from "Trips/Components/TripManagement";

const renderer = new ShallowRenderer();


describe('<TripManagement />', () => {
    const props = {
        trip: {id: 1},
    };

    test('renders correctly', () => {
        const result = renderer.render(<TripManagement {...props} />);
        expect(result).toMatchSnapshot();
    });
});