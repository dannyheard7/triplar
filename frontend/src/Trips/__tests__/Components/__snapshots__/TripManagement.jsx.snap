// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TripManagement /> renders correctly 1`] = `
<div
  className="trip-management"
>
  <Link
    replace={false}
    to="/trips/1/edit"
  >
    <button
      className="btn btn-light"
    >
      Edit
    </button>
  </Link>
  <Link
    replace={false}
    to="/trips/1/delete"
  >
    <button
      className="btn btn-danger"
    >
      Delete Trip
    </button>
  </Link>
</div>
`;
